Jenkinsfile.strict.si

```diff
@@ -36,6 +36,7 @@ node('py36') {
         } finally {
             junit allowEmptyResults: true, testResults: 'shakedown.xml'
             archive includes: "**/diagnostics.zip"
+            archive includes: "**/sandbox_.tar.gz"
```

Comment:
is this correct? I would expect something like `sandbox_*.tar.gz`

Reply:
Good catch! That's a typo